src/e2e/specs/landing.spec.ts

```diff
@@ -23,36 +26,32 @@ test.describe("Landing Page element verification", () => {
     const links = landingPage.links();
 
     // verify the actual hrefs in the footer matches expected
-    await expect(async () => {
-      expect(await landingPage.MozillaFooterLogoLink.getAttribute("href")).toBe(
-        links.mozillaLogoUrl,
-      );
-      expect(await landingPage.AllBreachesLink.getAttribute("href")).toContain(
-        links.allBreachesUrl,
-      );
-      expect(await landingPage.FAQLink.getAttribute("href")).toContain(
-        links.FAQUrl,
-      );
-      expect(await landingPage.TermsLink.getAttribute("href")).toContain(
-        links.TermsUrl,
-      );
-      expect(await landingPage.GithubLink.getAttribute("href")).toBe(
-        links.GithubUrl,
-      );
-    }).toPass();
+    expect(await landingPage.MozillaFooterLogoLink.getAttribute("href")).toBe(
```

Comment:
As mentioned elsewhere, `.toHaveAttribute` can be used in this file as well.

Reply:
iirc i think the errors are pretty close in terms of getting an "expected" and "actually" read out, interesting, i actually thought the other syntax was easier to read, but i'll update this